app/assets/javascripts/app/views/notifications_badge_view.js

```diff
@@ -0,0 +1,118 @@
+// @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
+
+app.views.NotificationsBadge = app.views.Base.extend({
```

Comment:
I'd call this `app.views.NotificationDropdown` because it doesn't change the badge.